Add reference to Gitlab on About/contributor-guides/for-staff (#677)
publiccodenet/about: contributor-guides/for-staff.md

* Attend morning standup ritual that is in the team calendar
* Attend sprint retrospective & planning - review the kanbans beforehand
* Tentoo NMBRS ESS (employee self service) mobile app - to [request leave](../activities/staff-management/leave.md), [file expenses](../activities/staff-management/expense.md) and see your payslips

> * if you already have a Gitlab account you can [join the Foundation for Public Code group](https://gitlab.com/publiccodenet) although we don't do any work on GitLab.

Reviewer: Inconsistent brand name :)

```suggestion
* if you already have a GitLab account you can [join the Foundation for Public Code group](https://gitlab.com/publiccodenet) although we don't do any work on GitLab.
```

> * if you already have a Gitlab account, [join the Foundation for Public Code group](https://gitlab.com/publiccodenet) - we only use this for our codebase stewardship work, so it may not be relevant to your role

Author: ```suggestion
* if you already have a Gitlab account, [join the Foundation for Public Code group](https://gitlab.com/publiccodenet) (note: we primarily work on Github)
```

Reviewer: I'd suggest something stronger. As we do no work on GitLab.

```suggestion
* if you already have a Gitlab account you can [join the Foundation for Public Code group](https://gitlab.com/publiccodenet) although we don't do any work on GitLab.
```
